src/index.js

```diff
@@ -122,6 +122,14 @@ const definePropertiesFor = (WebComponent, mapping, onAfterSet) => {
 
 export function register(ReactComponent, tagName, mapping = {}) {
   const attributeNames = Object.keys(mapping);
+  const render = function() {
```

Comment:
If this function is not a method of the class, I would make it an arrow function and pass this as a parameter instead of suing call. Since our friend Douglas said that `this` sucks :-). I know that the function is entirely pure then, bu at least we are avoiding this inside the context of the function.